src/System.Private.Uri/tests/ExtendedFuctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj

```diff
@@ -2,11 +2,13 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <TestTFM>netcoreapp1.1</TestTFM>
+    <FilterToTestTFM>netcoreapp1.1</FilterToTestTFM>
```

Comment:
This project will only work in netcoreapp1.1 which is why I added them here, but I can also add it on the builds file if you think its best